### PR TITLE
Add hello-world exercise

### PR DESCRIPTION
**Pull request by lpil**
Hello! I wanted to learn purescript, but alas, there is no exercise course. I figure I may as well try and build one while learning.

### CODE REVIEW

File: exercises/hello-world/bower.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "output"
+  ],
+  "dependencies": {
+    "purescript-prelude": "^2.1.0",
+    "purescript-console": "^2.0.0",
+    "purescript-test-unit": "^10.0.1"
```
Comment (paf31) [inline exercises/hello-world/bower.json]: Move this to `devDependencies`?

File: exercises/hello-world/src/HelloWorld.purs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module HelloWorld where
+
+import Prelude
+import Data.Maybe (Maybe(Just, Nothing))
+
+helloWorld :: Maybe String -> String
```
Comment (paf31) [inline exercises/hello-world/src/HelloWorld.purs]: Maybe if it's commented out?

**Comment by paf31** [comment]
The PureScript code looks good to me 👍

File: exercises/hello-world/bower.json (hunk 1 above)
Comment (lpil) [inline exercises/hello-world/bower.json]: Thank you.

**Comment by lpil** [comment]
Thanks :D

File: exercises/hello-world/src/HelloWorld.purs (hunk 2 above)
Comment (lpil) [inline exercises/hello-world/src/HelloWorld.purs]: I think the question for me is whether the student should be working out the type sig themselves or whether we should provide it.

**Comment by paf31** [comment]
@kytrinyx CI is passing but I'm not sure if all of the config files here are correct. The code looks good though.

**Comment by lpil** [comment]
I think there should be a script that makes sure the examples pass the tests, but I don't think it's been written yet.

File: config.json [hunk 3]
```diff
@@ -3,9 +3,14 @@
...
   "active": false,
-  "test_pattern": "TODO",
-  "problems": [
-
+  "exercises": [
+    {
+      "slug": "hello-world",
+      "difficulty": 1,
+      "topics": [
+        "maybe"
```
Comment (kytrinyx) [inline config.json]: At first I thought this was a placeholder, until I realized that PureScript has a `maybe` :)

**Comment by kytrinyx** [comment]
@lpil Yeah, it would be great to get something that tests the tests.

@paf31 The config looks good to me. I'll merge this and deploy the tracks and you should be able to test it with

```
exercism fetch purescript hello-world
```

**Comment by lpil** [comment]
Great, thank you. :)

I guess I'll keep going!

**Pull request merged**